pop-cli-for-appchains/guides/running-on-paseo-locally.md

```diff
@@ -244,18 +247,18 @@ Make sure to edit your chain spec and:&#x20;
 
 * **add your account and session keys**
 * **specify the starting balance of specific accounts**
-* **add the account that will be the sudo account for your parachain**
+* **add the account that will be the sudo account for your chain**
```

Comment:
I am a bit skeptical about inviting users to modify the plain spec directly instead of having code in the project that gives them the expected genesis.

But I think this is a different conversation than what this PR involves.

Reply:
Yes out of scope I'm afraid